onionshare_gui/settings_dialog.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, common, onion, qtapp, config=False, local_only=False):
         self.autostart_timer_checkbox = QtWidgets.QCheckBox()
         self.autostart_timer_checkbox.setCheckState(QtCore.Qt.Checked)
         self.autostart_timer_checkbox.setText(strings._("gui_settings_autostart_timer_checkbox"))
-        autostart_timer_label = QtWidgets.QLabel(strings._("gui_settings_whats_this").format("https://github.com/micahflee/onionshare/wiki/Using-the-Startup-Timer"))
+        autostart_timer_label = QtWidgets.QLabel(strings._("gui_settings_whats_this").format("Using-the-Auto-Start-Timer"))
```

Comment:
The link should be `https://github.com/micahflee/onionshare/wiki/Using-the-Auto-Start-Timer` instead of just `Using-the-Auto-Start-Timer` -- clicking it doesn't do anything :)

Reply:
I'll go ahead and make this commit